File: CS/Sample/Models/NorthwindDataModel.cs
using System;
using System.Collections.Generic;
using System.Data;
using System.Data.OleDb;
using System.Linq;
using System.Web;

namespace Sample.Models {
    public class ConnectionRepository {
        public static OleDbConnection GetDataConnection() {
            OleDbConnection connection = new OleDbConnection();
            connection.ConnectionString = string.Format("Provider=Microsoft.Jet.OLEDB.4.0;Data Source={0}", HttpContext.Current.Server.MapPath("~/App_Data/Northwind.mdb"));
            return connection;
        }
    }
    public class NorthwindDataModel {
        public static DataTable GetProducts() {
            DataTable dataTableProducts = new DataTable();
            using (OleDbConnection connection = ConnectionRepository.GetDataConnection()) {
                OleDbDataAdapter adapter = new OleDbDataAdapter(string.Empty, connection);
                adapter.SelectCommand.CommandText = "SELECT * FROM [Products]";
                adapter.Fill(dataTableProducts);
            }
            return dataTableProducts;
        }

        public static DataTable GetCategories() {
            DataTable dataTableCategories = new DataTable();
            using (OleDbConnection connection = ConnectionRepository.GetDataConnection()) {
                OleDbDataAdapter adapter = new OleDbDataAdapter(string.Empty, connection);
                adapter.SelectCommand.CommandText = "SELECT * FROM [Categories]";
                adapter.Fill(dataTableCategories);
            }
            return dataTableCategories;
        }
    }
}

File: VB/Sample_VB/Models/NorthwindDataModel.vb
Imports System
Imports System.Collections.Generic
Imports System.Data
Imports System.Data.OleDb
Imports System.Linq
Imports System.Web

Public Class ConnectionRepository
    Public Shared Function GetDataConnection() As OleDbConnection
        Dim connection As New OleDbConnection()
        connection.ConnectionString = String.Format("Provider=Microsoft.Jet.OLEDB.4.0;Data Source={0}", HttpContext.Current.Server.MapPath("~/App_Data/Northwind.mdb"))
        Return connection
    End Function
End Class
Public Class NorthwindDataModel
    Public Shared Function GetProducts() As DataTable
        Dim dataTableProducts As New DataTable()
        Using connection As OleDbConnection = ConnectionRepository.GetDataConnection()
            Dim adapter As New OleDbDataAdapter(String.Empty, connection)
            adapter.SelectCommand.CommandText = "SELECT * FROM [Products]"
            adapter.Fill(dataTableProducts)
        End Using
        Return dataTableProducts
    End Function

    Public Shared Function GetCategories() As DataTable
        Dim dataTableCategories As New DataTable()
        Using connection As OleDbConnection = ConnectionRepository.GetDataConnection()
            Dim adapter As New OleDbDataAdapter(String.Empty, connection)
            adapter.SelectCommand.CommandText = "SELECT * FROM [Categories]"
            adapter.Fill(dataTableCategories)
        End Using
        Return dataTableCategories
    End Function
End Class